full/spec/src/main/asciidoc/index.asciidoc

```diff
@@ -23,18 +23,20 @@
 :version-label!:
 :sectanchors:
 :doctype: book
-:license: Apache License v2.0
+:license: Eclipse Foundation Specification License v1.0
```

Comment:
Same question here.

Reply:
I basically done what has been done and approved in OpenAPI 2.0. But happy to also change the headers

Reply:
Works for me.  :)